memory-free-bytes.tf

```diff
@@ -9,7 +9,7 @@ module "memory_free_bytes" {
   source = "git@github.com:kabisa/terraform-datadog-generic-monitor.git?ref=terraform-provider-3"
 
   name             = "System - Memory Free Bytes"
-  query            = "avg(${var.memory_free_bytes_evaluation_period}):min:system.mem.free{${local.memory_free_bytes_filter}} by {host} < ${var.memory_free_bytes_critical}"
+  query            = "avg(${var.memory_free_bytes_evaluation_period}):min:system.mem.usable{${local.memory_free_bytes_filter}} by {host} < ${var.memory_free_bytes_critical}"
   alert_message    = "Available memory on ${var.service} Node {{host.name}} has dropped below {{threshold}} and has {{value}} available"
```

Comment:
Would it not be better if we referred to free memory instead of available memory?
```
 Free memory on ${var.service} Node {{host.name}} has dropped below {{threshold}} and has {{value}} available"
```

Reply:
My mistake, available is correct.

Reply:
Ah I thought you were having a semantic discussion  there :P